# timestamp of the update
time stamp

# updated state
TrayState state

# The action that let to this update
uint8 UNKNOWN=0
uint8 RESERVATION=1
uint8 DERESERVATION=2
uint8 OCCUPATION=3
uint8 DEOCCUPATION=4

uint8 action

